src/components/workflow/workflow-utils/getStatus.js

```diff
@@ -0,0 +1,47 @@
+// Utility functions for extracting node and dataset status info
+
+/**
+ * Get node status and duration from nodesStatus object.
+ * @param {Object} nodesStatus
+ * @param {Object} node
+ * @returns {{nodeStatus: string|null, nodeDuration: number|null}}
+ */
+export function getNodeStatusInfo(nodesStatus, node) {
```

Comment:
hey I want to clarify over here; getNodeStatusInfo referes to KedroNode. 
and then there's getNodeStatusKeys.js which refers to GraphNode. 

because we use Node everywhere; is it possible to do some distinction here and call 'Task' for KedroNodes and 'Nodes' for GraphNodes